keyboards/electronlab/klor/keymaps/default/config.h

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2024 ElectronLab
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* The way how "handedness" is decided (which half is which),
+see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness
+for more options.
+*/
+
+#define EE_HANDS
```

Comment:
Agreed. Change is done and committed.
I have moved the default definition to MASTER_LEFT for an easier flashing and fail proof process. EE_HANDS would require a differential compilation which is not desired for a basic implementation. I have left the commented out options as this may also be changed depending of the hardware revision of the PCB with a predetermined pin number for handedness.